gerrychain/tree.py

```diff
@@ -373,62 +566,80 @@ def bipartition_tree(
     :param spanning_tree_fn: The random spanning tree algorithm to use if a spanning
         tree is not provided. Defaults to :func:`random_spanning_tree`.
     :type spanning_tree_fn: Callable, optional
-    :param weight_dict: A dictionary of weights for the spanning tree algorithm.
+    :param region_surcharge: A dictionary of surcharges for the spanning tree algorithm.
         Defaults to None.
-    :type weight_dict: Optional[Dict], optional
+    :type region_surcharge: Optional[Dict], optional
     :param balance_edge_fn: The function to find balanced edge cuts. Defaults to
         :func:`find_balanced_edge_cuts_memoization`.
     :type balance_edge_fn: Callable, optional
-    :param choice: The function to make a random choice. Can be substituted for testing.
-        Defaults to :func:`random.choice`.
+    :param choice: The function to make a random choice. Passed to ``balance_edge_fn``.
```

Comment:
can you specify that this is the random choice of nodes, not cuts?

Reply:
Done!